package/MDAnalysis/units.py

```diff
@@ -199,6 +199,8 @@
     'N_Avogadro': 6.02214129e+23,          # mol**-1
     'elementary_charge': 1.602176565e-19,  # As
     'calorie': 4.184,                      # J
+    'Boltzman_constant': 8.314462159e-3,   # KJ (mol K)**-1
+    'electric_constant': 5.526350e-3,      # As (Angstroms Volts)**-1
```

Comment:
Is there a reason we can’t just import from SciPy.constants now? Less code to maintain. In fact I’d sooner just import from scipy in the actual module rather than us maintain our own list of constants

Reply:
Until very recently, scipy was an optional dependency. I guess it would be fine now.

Reply:
I’m also preferring using the scipy constants directly. The question for this PR is: Should we keep it like this for now and update it later?
Should we introduce scipy.constants for the new constants here? Or first adjust the units.py?

Reply:
I would say that changing how the constants are handled is out of scope for this PR.